### ISSUE TITLE
npm package for v3.1.0 seems to contain older version

### ISSUE THREAD

**Issue by webmatze** [wbuchwalter/ng-redux#37]
I tried to install **ng-redux** version 3.1.0 using **npm**:

    $ npm install --save ng-redux

But the installed package code still contains references to the complete `lodash` package. Even after clearing the **npm** cache. So I have to manually include the `lodash` to my project as the package is missing in the **ng-redux** package dependencies.

**Comment by wbuchwalter** [comment]
Hey Mathias,
I can't reproduce your issue.
I also checked the source again and didn't find any reference to the complete lodash package.
Can you check which part of ng-redux throws `undefined` before manually including lodash?
Alternatively, can I see your code somewhere so I can reproduce that?
Thanks!

**Comment by webmatze** [comment]
Yes, you can test it with this repository: [webmatze/simplest-angular-redux-example](https://github.com/webmatze/simplest-angular-redux-example).

When I clone it into a new directory, remove the dependency to **lodash** in the ``package.json`` and run `npm install` followed by `npm start` it throws the following exeption:

```
> simplest-angular-redux-example@0.0.1 start /Users/webmatze/Workspace/forks/simplest-angular-redux-example
> watchify index.js -g uglifyify --debug --extension=js -v -o 'exorcist bundle.js.map > bundle.js' & python -m SimpleHTTPServer

Serving HTTP on 0.0.0.0 port 8000 ...
Error: Cannot find module 'lodash' from '/Users/webmatze/Workspace/forks/simplest-angular-redux-example/node_modules/ng-redux/lib/components'
Error: Cannot find module 'lodash' from '/Users/webmatze/Workspace/forks/simplest-angular-redux-example/node_modules/ng-redux/lib/components'
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: write after end
    at writeAfterEnd (/Users/webmatze/Workspace/forks/simplest-angular-redux-example/node_modules/watchify/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:198:12)
    at DestroyableTransform.Writable.write (/Users/webmatze/Workspace/forks/simplest-angular-redux-example/node_modules/watchify/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:243:5)
    at DestroyableTransform.Writable.end (/Users/webmatze/Workspace/forks/simplest-angular-redux-example/node_modules/watchify/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:474:10)
    at Readable.<anonymous> (/Users/webmatze/Workspace/forks/simplest-angular-redux-example/node_modules/watchify/bin/cmd.js:47:16)
    at Readable.emit (events.js:129:20)
    at Labeled.<anonymous> (/Users/webmatze/Workspace/forks/simplest-angular-redux-example/node_modules/browserify/node_modules/read-only-stream/index.js:28:44)
    at Labeled.emit (events.js:107:17)
    at Labeled.<anonymous> (/Users/webmatze/Workspace/forks/simplest-angular-redux-example/node_modules/browserify/node_modules/labeled-stream-splicer/node_modules/stream-splicer/index.js:130:18)
    at Labeled.emit (events.js:129:20)
    at Deps.<anonymous> (/Users/webmatze/Workspace/forks/simplest-angular-redux-example/node_modules/browserify/node_modules/labeled-stream-splicer/node_modules/stream-splicer/index.js:130:18)
```

Then I checked the code of ``connector.js`` in the directory ``cd node_modules/ng-redux/lib`` and found the reference to **lodash**:

```
var _lodash = require('lodash');
```

But if I look into the **package.json** of **ng-redux** I can find the following version 
```
"version": "3.1.0",
```

I even looked into the npm cached version and found the same code:

```
~/.npm/ng-redux/3.1.0/package.tgz
```

I also deleted all the contents of ``~/.npm/ng-redux`` and reinstalled **ng-redux** but that did not removed the problem.

**Comment by wbuchwalter** [comment]
You're totally right, sorry for the hassle, will fix this ASAP.
Thanks.

**Comment by wbuchwalter** [comment]
Should be fixed now with `3.1.1`.
Thanks!

**Comment by webmatze** [comment]
Yes, I can confirm that. It's fixed! Thank you. Great work!
